pandas/plotting/_core.py

```diff
@@ -1805,10 +1807,7 @@ def _get_plot_backend(backend=None):
             ) from None
 
         _backends["matplotlib"] = module
-
-    if backend in _backends:
-        return _backends[backend]
+        return module
```

Comment:
As `_get_plot_backend` is necessarily the one importing the `matplotlib` backend, I was wondering whether it made more sense to have only this function populating `_backends`, and have `_find_backend` only return a module, to avoid having `_backends` getting modified in several places.

Reply:
Resolved with e84b305